blivet/flags.py

```diff
@@ -77,6 +77,9 @@ def __init__(self):
         # (so far only for LUKS)
         self.discard_new = False
 
+        # wheteher cdroms should be made protected during devicetree population
+        self.protect_cdroms = True
```

Comment:
This should default to False.

Reply:
Ah, sure.

Reply:
@dwlehman I fixed the default value and changed the approach a bit (see below)